tests/test_other.py

```diff
@@ -3336,7 +3336,6 @@ def test_warn_unaligned(self):
   return 0;
 }
 ''')
-    output = run_process([PYTHON, EMCC, 'src.cpp', '-s', 'WASM=0', '-s', 'WARN_UNALIGNED=1'], stderr=PIPE)
```

Comment:
Why remove this?

Reply:
`output` is overwritten on the next line. so it's pointless, except I guess as checking compilation doesn't fail. but it seems redundant with the next line.